packages/flutter/test/material/checkbox_list_tile_test.dart

```diff
@@ -171,18 +174,56 @@ void main() {
       ),
     );
 
-    expect(tester.widget<Checkbox>(find.byType(Checkbox)).value, null);
+    expect(tester.widget<Checkbox>(find.byType(Checkbox)).value, false);
+
+    // tap checkbox when disable tristate
```

Comment:
These comments are helpful. Wordsmithing: Tap the checkbox when tristate is disabled.

Reply:
Done.